pages/content/ch5/fields.mdx

```diff
@@ -335,3 +335,56 @@ Because we were able to eliminate the possibilities $1, 2, 3, 4, 6, 8, 12$ as or
 </div>
 </Problem>
 
+## Exercises
+
+<Problem title="MGO" source="Philipp Barski, Yannick Funke" difficulty={3} relevance={3}>
+    <div label="question">
+    What is the order of $\mathrm{GF}(p)[x]_{x^m}^*$? Prove your answer.
+    </div>
+    <div label="hint">
+        Use the fact that $\mathrm{GF}(p)[x]_{x^m}^* = \{ a \in \mathrm{GF}(p)[x]_{x^m} \mid \gcd(a, x^m)=1 \}$.
+    </div>
+    <div label="hint">
+        Show that $\gcd(a, x^m) \neq 1 \iff x | a$.
+    </div>
+    <div label="hint">
+        Every polynomial in $\mathrm{GF}(p)[x]$ can be factored uniquely into irreducible polynomials up to taking associates (constant factors).
+    </div>
+    <div label="answer">
+        Define $R = \mathrm{GF}(p)[x]_{x^m}$.
+
+        In the following we will show that, for any $a(x) \in R$ we have that $\gcd(a(x), x^m) \neq 1 \iff x|a(x)$.
+
+        Let $a(x) \in R$ be arbitrary.
+
+        We first show $\implies$: Assume $\gcd(a(x), x^m) \eqqcolon g(x) \neq 1$. Then we have that $g(x) \vert x^m$, by definition. This implies that we have $g(x)b(x) = x^m$ for some $b(x) \in \mathrm{GF}(p)[x]$. Note that by definition $g(x)$ has to be monic and since $g(x) \neq 1$ we have that $g(x)$ is not constant. Note that by Theorem 5.27 we need that $g(x) = k \cdot x^d$ with $0 \leq d \leq m$ and $k \in \mathrm{GF}(p)$ since $x$ is irreducible in $\mathrm{GF}(p)[x]$ and we have that degrees of polynomials are added for polynomials on integral domains and a field is in particular an integral domain. As $g(x)$ is monic and not constant we have that $k = 1$ and $1 \leq d \leq m$. Note that this means that $x \vert g(x)$. Since $g(x) \vert a(x)$ by transitivity of $\vert$ we obtain that $x \vert a(x)$.
```

Comment:
> Note that by Theorem 5.27 we need that $g(x) = k \cdot x^d$ with $0 \leq d \leq m$ and $k \in \mathrm{GF}(p)$ since $x$ is irreducible in $\mathrm{GF}(p)[x]$

Das sollte elaboriert werden. Ist nicht so offensichtlich, dass keine Zwischenschritte braucht. Können wir aus meiner Version kopieren. Da kam das noch vor.

Reply:
Sagst du bei Primzahlen auch, dass dort elaboriert werden sollte? z.B.: ab = p^3. Dann impliziert das auf Grund der eindeutigen Primfaktorzerlegung bis auf units, dass a = k * p^i, wobei k \in \{-1,1\}, 0 \leq i \leq 3.

Reply:
Ja, ok. Können wir vllt Theorem 5.27 kurz erläutern? Sonst muss man es erst nachschlagen, um den Beweis zu verstehen.

Reply:
Und du kannst das k streichen, da g monisch sein muss